### ISSUE TITLE
Don't prompt to install ipykernel when request comes from GH Codespaces, install it directly

### ISSUE THREAD

**Issue by luabud** [microsoft/vscode-jupyter#4882]
This was discussed in the GH Codespaces repo. When users request to open the interactive window or to open a Jupyter notebook and ipykernel isn't installed, instead of prompting them to install it, it could just skip the prompt and move with the installation directly. Because this is running on a remote machine, I believe the concerns that lead us to prompt before installing locally are different.

**Comment by DonJayamanne** [comment]
@SiddhanthUnnithan @greazer  Should we do the same for AML Compute?

**Comment by joyceerhl** [comment]
I created a GitHub codespace, verified that ipykernel was not installed, installed the VSIX from #4985 artifacts and the Python extension from the marketplace, and opened a notebook. When I ran a cell, ipykernel got installed without my needing to acknowledge the prompt.

![image](https://user-images.githubusercontent.com/30305945/109739652-cc61f680-7b7e-11eb-85f5-33ac5df22954.png)

IMO it would be better if the user never saw the progress indicator at all, but assuming this is what the PMs wanted, closing as validated.
